amida-tech/indaba-client
# Feature/INBA-454 Address Fields

#### What's this PR do?
1. Add a component to select recipients of a message
1. Render friendly names for message recipients and senders

#### Related JIRA tickets:
[INBA-454](https://jira.amida-tech.com/browse/INBA-454)

#### How should this be manually tested?
1. Start a new message
1. Check that you can search for and select a user to send the message to
1. Check that the selected recipient is added to the recipient list
1. Check that the from: field renders a friendly name for the sender, and is not editable
1. Check that sending messages generates entries in the messaging service database with users' emails as the owners, to, and from columns

#### Any background context you want to provide?
This branch sits on the #226 branch

#### Screenshots (if appropriate):
